services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js

```diff
@@ -41,18 +35,24 @@ import {
   fetchApplicationDelay,
   fetchImportedNoticeOfWorkApplication,
 } from "@common/actionCreators/noticeOfWorkActionCreator";
-import CustomPropTypes from "@/customPropTypes";
-import { modalConfig } from "@/components/modalContent/config";
 import { openModal, closeModal } from "@common/actions/modalActions";
 import {
   getDraftPermitForNOW,
   getDraftPermitAmendmentForNOW,
 } from "@common/selectors/permitSelectors";
+import { PERMIT_AMENDMENT_TYPES } from "@common/constants/strings";
+import { getDocumentContextTemplate } from "@/reducers/documentReducer";
+import {
+  generateNoticeOfWorkApplicationDocument,
+  fetchNoticeOfWorkApplicationContextTemplate,
+} from "@/actionCreators/documentActionCreator";
+import CustomPropTypes from "@/customPropTypes";
+import { modalConfig } from "@/components/modalContent/config";
 import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
 import * as Permission from "@/constants/permissions";
 import * as route from "@/constants/routes";
 import NOWTabHeader from "@/components/noticeOfWork/applications/NOWTabHeader";
-import { PERMIT_AMENDMENT_TYPES } from "@common/constants/strings";
+import LinkButton from "@/components/common/buttons/LinkButton";
 import { APPLICATION_PROGRESS_TRACKING } from "@/constants/NOWConditions";
```

Comment:
why was this removed/added?

Reply:
This was autoformatted, if you noticed the imports are moved, but are still there. Going to try to remove that change.

Reply:
phew okay! disregard.